rules.mk

```diff
@@ -37,7 +37,7 @@ export PATH
 endif
 
 # parses tools.go and returns the tool name prefixed with bin/
-toolsBins := $(addprefix bin/,$(notdir $(shell awk -F'"' '/^\s*_/ {print $$2}' tools.go)))
+toolsBins := $(addprefix bin/,$(notdir $(shell awk -F'"' '/^\s*/ {print $$2}' tools.go)))
```

Comment:
follow up: I checked the commit that did this but didn't see the underlying issue. If we have to keep the `_` out it would be good to have the cause in the message.

Reply:
https://github.com/tinkerbell/boots/pull/229/commits/c407c9e5277ba73cefdb956d678998ddd84ab6ea

_"Get make gen working on MacOS"_

This change doesn't seem to be working anymore. I just tested on another Mac. I will need to revisit this.

Reply:
from testing this. the `awk` line with the `_` isn't working for me at all. tested in a Linux container and on a mac. once the `_` is removed, then it works on both.